client/src/components/layout/MovieTile.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import '../../assets/scss/styles.scss'
+
+const MovieTile = ( { movie }) => {
+
+    return (
+        <div className="grid-container"> 
```

Comment:
there are 2 top level `div`'s in this component - this one with `grid-container` should be ok to remove as it likely isn't providing much in terms of adding another `div`, and `grid-container` just adds slight spacing on all sides of the element
https://get.foundation/sites/docs/xy-grid.html#grid-container